### PR TITLE
DOCS-1520: OS 3.26.1 publishing

### PR DESCRIPTION
**Pull request by girishshankaran**
This is the PR for publishing the OS 3.26.1 patch release.

Product Version(s):
Calico 3.26.1

Issue:
<https://tigera.atlassian.net/browse/DOCS-1520>

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by girishshankaran** [comment]
@ctauchen While I'm waiting for the release notes, can you take a look at the other things?

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+12 Jun 2023
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx]: Update

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+#### Bug fixes
+
+ - Fix an issue with Openshift violations of the restricted Pod Security profile [calico #7768](https://github.com/projectcalico/calico/pull/7768) (@MichalFupso)
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx]:
```suggestion
 - Fix an issue with OpenShift violations of the restricted pod security profile [calico #7768](https://github.com/projectcalico/calico/pull/7768) (@MichalFupso)
```

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ - Calico's integration code for OpenStack has been updated for OpenStack Yoga. [calico #7746](https://github.com/projectcalico/calico/pull/7746) (@nelljerram)
+ - eBPF: Fix applyOnforward=false in global policies [calico #7724](https://github.com/projectcalico/calico/pull/7724) (@tomastigera)
```
Comment (ctauchen) [inline calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx]:
```suggestion
 - eBPF: Fix `applyOnforward=false` in global policies [calico #7724](https://github.com/projectcalico/calico/pull/7724) (@tomastigera)
```

**Review by ctauchen — APPROVED** [review]
LGTM, with small changes.

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx (hunk 1 above)
Comment (girishshankaran) [reply calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx]: Done

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx (hunk 2 above)
Comment (girishshankaran) [reply calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx]: Done

File: calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx (hunk 3 above)
Comment (girishshankaran) [reply calico_versioned_docs/version-3.26/_includes/release-notes/_v3.26.1-release-notes.mdx]: Done

**Comment by ctauchen** [comment]
See #749

**Pull request closed without merging**